src/board/internal.tsx

```diff
@@ -47,11 +48,13 @@ export function InternalBoard<D>({
   const containerRef = useMergeRefs(containerAccessRef, containerQueryRef);
   const itemContainerRef = useRef<{ [id: ItemId]: ItemContainerRef }>({});
 
+  const isRtl = getIsRtl(containerAccessRef.current);
```

Comment:
Just curious why rtl detection is based on a specific dom element since it has already been detected on `pages/app/page.tsx` level? Wouldn't it be more appropriate to introduce a new context for that purpose, where the value is provided at the `pages/app/page.tsx` level and then consumed on other pages?

Reply:
The page is the consumer code while the board is ours. We don't want the customers to pass RTL explicitly. It might make sense to have a simpler mechanism to propagate isRtl but this should be then done holistically for both components and board-components. Also one known issue with getIsRtl is that it requires an element reference and hence is always false for the very first call.